WD-Firmware/psp/src/psp_interrupts.c

```diff
@@ -0,0 +1,295 @@
+/* 
+* SPDX-License-Identifier: Apache-2.0
+* Copyright 2019 Western Digital Corporation or its affiliates.
+* 
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+* 
+* http:*www.apache.org/licenses/LICENSE-2.0
+* 
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+/**
+* @file   psp_interrupts.c
+* @author Ronen Haen
+* @date   20.05.2019
+* @brief  The file supllies interrups services such as  - registration, default handlers and indication
+*         whether we are in interrpu context
+* 
+*/
+
+/**
+* include files
+*/
+#include "psp_types.h"
+#include "psp_api.h"
+
+/**
+* definitions
+*/
+
+/* The stack used by interrupt service routines */
+#ifdef D_ISR_STACK_SIZE_WORDS
```

Comment:
"_WORDS" - remove it it is bytes

Reply:
https://github.com/westerndigitalcorporation/riscv-fw-infrastructure/pull/12/commits/6c26f8aba68f07621c53b564702cbab8a7d91dd1